utils/docker/Dockerfile.maldet.el.8

```diff
@@ -0,0 +1,169 @@
+    fi
+
+ARG BULLSEYE
+RUN if [ "x$BULLSEYE" != "x" ]; then \
+      curl ${JENKINS_URL}job/daos-stack/job/tools/job/master/lastSuccessfulBuild/artifact/bullseyecoverage-linux.tar \
+        --retry 10 --retry-max-time 60 --silent --show-error -o bullseye.tar; \
+      mkdir -p bullseye; \
+      tar -C bullseye --strip-components=1 -xf bullseye.tar; \
+      pushd bullseye; \
+        ./install --quiet --key "${BULLSEYE}" --prefix /opt/BullseyeCoverage; \
+      popd; \
+      rm -f bullseye.tar; \
+      rm -rf bullseye; \
+    fi
+
+USER daos_server:daos_server
+
+# Setup a python venv so that python packages can be installed locally.
+RUN python3 -m venv /home/daos/venv
+ENV PATH=/home/daos/venv/bin:$PATH
+ENV VIRTUAL_ENV=/home/daos/venv/
+
+# Install latest versions of python tools.
+# wheel is needed first to avoid a warning when installing pyyaml.
+COPY requirements.txt .
+RUN python3 -m pip --no-cache-dir install --upgrade pip && \
+    python3 -m pip --no-cache-dir install wheel && \
+    python3 -m pip --no-cache-dir install -r requirements.txt
+
+WORKDIR /home/daos/pre
+COPY --chown=daos_server:daos_server SConstruct .
+COPY --chown=daos_server:daos_server site_scons site_scons
+COPY --chown=daos_server:daos_server utils/build.config utils/
+
+# Control what to build.  By default Dockerfiles build everything to allow for
+# ease-of-use for users, however in CI everything is turned off and then
+# selectively enabled.  Turning off any step disables all future steps.
+ARG DAOS_DEPS_BUILD=yes
+ARG DAOS_KEEP_BUILD=no
+ARG DAOS_TARGET_TYPE=release
+
+# Now do an update to ensure software is up to date for the deps build.  If the
+# src hasn't changed then this won't do anything, but if it has then we want to
+# ensure that latest dependencies are used.
+USER root:root
+RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                                               \
+        dnf upgrade --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury && \
+        dnf clean all;                                                                 \
+    }
+USER daos_server:daos_server
+
+ARG DEPS_JOBS=1
+
+RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                            \
+        scons --build-deps=yes --jobs $DEPS_JOBS PREFIX=/opt/daos   \
+              TARGET_TYPE=$DAOS_TARGET_TYPE --deps-only &&          \
+        ([ "$DAOS_KEEP_BUILD" != "no" ] || /bin/rm -rf build *.gz); \
+    }
+USER root:root
+
+# force an upgrade to get any newly built RPMs, but only if CACHEBUST is set.
+ARG CACHEBUST
+RUN [ -z "$CACHEBUST" ] || {                                                           \
+        dnf upgrade --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury && \
+        dnf clean all;                                                                 \
+    }
+
+ARG DAOS_KEEP_SRC=no
+# Remove local copy
+RUN [ "$DAOS_KEEP_SRC" != "no" ] || rm -rf /home/daos/*
+
+ARG BUILD_URL
```

Comment:
```suggestion
```

Reply:
Still need most of that.   What little that I do not need would invalidate the cache layers if I removed them.
Don't need bullseye, or the python stuff, or deps_build steps.

Need the user setup for SUDO, having dependencies installed, and updating all the packages.